botfront/imports/lib/utils.js

```diff
@@ -163,9 +163,9 @@ if (Meteor.isServer) {
                 const axiosJson = axios.create();
                 addLoggingInterceptors(axiosJson, appMethodLogger);
                 // 400mb
-                const maxContentLength = 400000000;
+                const maxContentLength = maxBodyLength = 400000000;
```

Comment:
```suggestion
                const maxContentLength = 400000000;
                const maxBodyLength = 400000000;
```

Reply:
This would be easier to understand, otherwise, it's looking good, I'll merge once that change is made

Reply:
Done.